### PR TITLE
Add Duration.Sol (day on Mars)

### PR DESCRIPTION
**Pull request by octastylos-pseudodipteros**
Closes issue #1346 by adding sol, a Mars day, which lasts 88775 seconds.

### CODE REVIEW

**Comment by octastylos-pseudodipteros** [comment]
Hello @angularsen, sorry to bother, but I would like to ask if you could check out my pull request, thank you

**Review by angularsen — CHANGES_REQUESTED** [review]

File: Common/UnitDefinitions/Duration.json [hunk 1]
```diff
@@ -154,6 +154,21 @@
...
         }
       ]
+    },
+    {
+      "SingularName": "Sol",
+      "PluralName": "Sols",
+      "BaseUnits": {
+        "T": "Sol"
+      },
+      "FromUnitToBaseFunc": "{x} * 88775",
```
[the remark this answers is not in the document]
Comment (angularsen) [reply Common/UnitDefinitions/Duration.json:164]: @octastylos-pseudodipteros Thanks for pinging, it seems my review was not submitted and not visible to you. Please take a look.

**Comment by octastylos-pseudodipteros** [comment]
@angularsen thank you for your reply, the conversion value is now corrected.

**Comment by angularsen** [comment]
Thanks, nugets are on the way.

[Release UnitsNet/5.56.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F5.56.0)

[Release UnitsNet/6.0.0-pre011 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F6.0.0-pre011)

**Comment by angularsen** [comment]
The nuget was delayed due to a mistake, should be out within half an hour or so.

**Pull request merged**